mathics/core/expression.py

```diff
@@ -430,6 +613,19 @@ def filter(self, head, cond, evaluation):
         # faster equivalent to: Expression(head, [element in self.leaves if cond(element)])
         return structure(head, self, evaluation).filter(self, cond)
 
+    def filter_elements(self, head_name):
```

Comment:
This function is never used.

Reply:
Ok - good. It is been removed then. 

It is interesting in that when I looked at the v0.5 released code where the evaluate() method was very different, that code was in there. I especially remember he a TODO in there about how element sorting needed to be done. 

I guess that when the 2nd maintainer added that this was orphaned and didn't get removed.